src/providers/oauth-provider.js

```diff
@@ -11,6 +11,7 @@ var defaults = {
   clientId: null,
   clientSecret: null,
   grantPath: '/oauth2/token',
+  grantType: 'password',
```

Comment:
It is not as simple as that. Mainly because, initially, we implemented the access token getter based on the [Resource Owner Password Credentials Grant](https://tools.ietf.org/html/rfc6749#section-4.3), which along with the data (password and username) the required value for the `grant_type` should be set to `password`. By changing the `grant_type` there should be the possibility to extend the `data` object. Instead of just changing the `grant_type` on the configuration, the method `getAccessToken` could be changed, by passing a `data` property instead of `user`, as explained at #43. IMHO it would be a better option instead.